packages/story-editor/src/components/canvas/framesLayer.js

```diff
@@ -57,28 +58,44 @@ const DesignSpaceGuideline = styled.div`
   visibility: ${({ isVisible }) => (isVisible ? 'visible' : 'hidden')};
 `;
 
-function FramesLayer() {
-  const { currentPage, isAnimating } = useStory((state) => ({
-    currentPage: state.state.currentPage,
-    isAnimating: [
-      STORY_ANIMATION_STATE.PLAYING,
-      STORY_ANIMATION_STATE.SCRUBBING,
-    ].includes(state.state.animationState),
+const DesignSpaceGuideline = function DesignSpaceGuidline() {
```

Comment:
🤔 

Why not just `function DesignSpaceGuideline() {`?

Not to mention the typo in `Guidline`

Reply:
ah I was trying something where I was wrapping this in a `forwardRef`, then forgot to remove this when I removed the `forwardRef` bit.

Thanks for pointing out. will clean up